tests/integration/fixtures/mini_sentry.py

```diff
@@ -105,12 +105,14 @@ def check_challenge():
 
     @app.route("/api/666/store/", methods=["POST"])
     def store_internal_error_event():
+        try:
+            data = json.loads(flask_request.data)
+        except json.JSONDecodeError:
+            data = flask_request.data
         sentry.test_failures.append(
             (
                 "/api/666/store/",
-                AssertionError(
-                    "Relay sent us event: %s" % get_error_message(flask_request.data)
-                ),
+                AssertionError("Relay sent us event:\n{}".format(pformat(data))),
```

Comment:
Is there a problem with `get_error_message`? This was specifically introduced to render readable errors rather than endless JSON that contains no useful information.

Also, this means that `get_error_message` is dead code now.

Reply:
You can see an example of that in this very build actually:
https://travis-ci.com/github/getsentry/relay/jobs/361775792#L1549

The intention is to *only* print the error message(s), which is extracted by `get_error_message` from the event payload, and then formatted by `format_failures()`.

Reply:
oh sorry :(. Thanks for catching.  I was working from a version where `get_error_message()` didn't exist yet.  I'll put it back.